.github/workflows/example-basic-pnpm.yml

```diff
@@ -21,13 +21,17 @@ jobs:
         # See https://github.com/pnpm/action-setup
       - name: Install pnpm
         uses: pnpm/action-setup@v4
+        # Optional when there is a `packageManager` field in the `package.json`.
+        # See https://github.com/pnpm/action-setup?tab=readme-ov-file#version
```

Comment:
https://nodejs.org/api/corepack.html lists this feature as experimental and not suitable for production.

> Stability: 1 - Experimental. The feature is not subject to [semantic versioning](https://semver.org/) rules. Non-backward compatible changes or removal may occur in any future release. Use of the feature is not recommended in production environments.

Reply:
Interesting. However since the action has documentation on supporting it, they will still lookup the information from there, even if `packageManager` is not an official property anymore, right?